src/types/interface.ts

```diff
@@ -49,6 +51,54 @@ export interface InterfaceTypeOptions {
     // TODO: allowUnknownProperties?: boolean; // default: false
 }
 
+export interface InterfaceTypeOptionsWithPartial extends InterfaceTypeOptions {
+    /** Mark all properties as optional in this type. */
+    partial?: boolean;
+}
+
+export interface InterfaceMergeOptions {
+    /** The optional name for the type, uses a default TypeScript-like name if no name is given. */
+    name?: string;
+
+    /**
+     * Suppress the error about existing custom validations on one of the types that is being merged.
```

Comment:
This doesn't just 'suppress the error' anymore. You can merge a type without its validations with this option, even if it would not error otherwise.